tests/requirements-tools.txt

```diff
@@ -12,4 +12,4 @@ git+https://github.com/xoviat/pytest-plugins.git@common_setup#egg=pytest-profili
 
 psutil  # Better subprocess alternative with implemented timeout.
 
-https://github.com/erocarrera/pefile/archive/develop.zip  # contains changes not yet released
+git+https://github.com/xoviat/pefile.git@patch-1  # contains optimizations to be merged
```

Comment:
Optimization are merges meanwhile, see https://github.com/pyinstaller/pyinstaller/issues/1920#issuecomment-310918505. If we remove the bundled pefile, please change back to `…/erocarrera/pefile/archive/develop.zip`